frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/ActionUtils.java

```diff
@@ -26,17 +31,17 @@ public static IBranchLayoutWriter getBranchLayoutWriter(AnActionEvent anActionEv
   }
 
   public static Option<String> getCurrentBranchNameIfManaged(AnActionEvent anActionEvent) {
-    return getGitMacheteRepository(anActionEvent)
-        .flatMap(repo -> repo.getCurrentBranchIfManaged())
-        .map(branch -> branch.getName());
+    return getCurrentMacheteBranch(anActionEvent).map(branch -> branch.getName());
+  }
+
+  public static Option<BaseGitMacheteBranch> getCurrentMacheteBranch(AnActionEvent anActionEvent) {
```

Comment:
```suggestion
  public static Option<BaseGitMacheteBranch> getCurrentMacheteBranchIfManaged(AnActionEvent anActionEvent) {
```
?

Reply:
Fine, but I don't like this suffix in this scope. At git machete repository level it makes sense indeed. Here, talking frontendActions, utils in particular everything is an option. IMHO returning an option indicates some `if`s somewhere that decide about the result presence. it is kind-of default and natural.